static/js/admin/vue-permissions-editor/src/components/embargo.vue

```diff
@@ -170,22 +161,23 @@
 
                 if (date_filled && regex_match && isFuture(date_parts)) {
                     this.embargo_type = 'custom';
-                    this.$emit('error-msg', '');
-                    this.embargo_ends_date = this.custom_embargo_date;
-                    this.$emit('embargo-info', {
-                        embargo: this.embargo_ends_date,
-                        skip_embargo: false
+                    this.error_msg = '';
+                    this.$store.commit('setEmbargoInfo', {
+                        embargo: this.custom_embargo_date,
+                        skipEmbargo: false
                     });
                 } else if (date_filled && !regex_match) {
-                    this.$emit('error-msg', 'Please enter a valid date in the following format YYYY-MM-DD');
+                    this.error_msg = 'Please enter a valid date in the following format YYYY-MM-DD';
```

Comment:
Are we using `this.error_msg` for anything aside from passing it along to the alertHandler? Wondering if we don't need it as an instance variable.

Reply:
I originally didn't have it as a variable, but it made it pretty much impossible to update a couple of the tests, as they were checking the message response.